VAMobile/src/api/claimsAndAppeals/submitClaimDecision.tsx

```diff
@@ -0,0 +1,41 @@
+import { SetDataOptions, useMutation, useQueryClient } from '@tanstack/react-query'
+import { DateTime } from 'luxon'
+
+import { ClaimData, ClaimDecisionResponseData } from 'api/types'
+import { post } from 'store/api'
+import { logNonFatalErrorToFirebase } from 'utils/analytics'
+import { isErrorObject } from 'utils/common'
+
+import { claimsAndAppealsKeys } from './queryKeys'
+
+/**
+ * Notifes VA to make a claim decision
+ */
+
+const submitClaimDecision = (claimID: string) => {
+  return post<ClaimDecisionResponseData>(`/v0/claim/${claimID}/request-decision`)
+}
+
+/**
+ * Returns a mutation to make a claim decision
+ */
+export const useSubmitClaimDecision = (claimID: string) => {
+  const queryClient = useQueryClient()
+  const claimData = queryClient.getQueryData([claimsAndAppealsKeys.claim, claimID]) as ClaimData
+  claimData.attributes.waiverSubmitted = true
+
+  return useMutation({
+    mutationFn: submitClaimDecision,
+    onSuccess: () => {
+      const setDataOptions: SetDataOptions = {
+        updatedAt: DateTime.now().toMillis(),
+      }
```

Comment:
Is this needed? By default, `updatedAt` is already set to `Date.now()`

Reply:
yes it is needed. It actually wasn't set to Date.now() as it refreshed right afterwards on Rachael in testing

Here is the github issue referencing the issue

https://github.com/TanStack/query/issues/1296